include/open62541/types.h

```diff
@@ -424,6 +424,7 @@ UA_NodeId_parse(UA_NodeId *id, const UA_String str);
 UA_INLINABLE(UA_NodeId
              UA_NODEID(const char *chars), {
     UA_NodeId id;
+    memset(&id, 0, sizeof(UA_NodeId));
```

Comment:
This one is good.
But I'd rather move it into UA_NodeId_parse.
So it comes automatic in more cases.

Reply:
@jpfr: In the function parse_nodeid the id is already set to UA_NODEID_NULL, so this might already be enough initalization for the function UA_NodeId_parse?

Reply:
You are right. That is enough.

Reply:
Should I remove the memset in line 427?

Reply:
Yes.

Reply:
Removed memset in line 427.